### ISSUE TITLE
Generator: Set up SonarQube

### ISSUE THREAD

**Issue by blackfalcon** [AlaskaAirlines/WC-Generator#376]
## Is your feature request related to a problem? Please describe.

The security standard for Alaska is to use SonarQube

## Describe the solution you'd like

The specification of work is as follows: 

1. Remove support for Synk, it is no longer needed
2. Remove support for Github's CodeQL testing tools
3. INSTALL SonarQube

The recommended solution is to use the [SonarQube Github action](https://github.com/marketplace/actions/sonarqube-scan). Reach out to Alaska security to obtain the host: `${{ secrets.SONARQUBE_HOST }}` and login: `${{ secrets.SONARQUBE_TOKEN }}` auth tokens/secrets for the repos. 

**_Test the workflow with an existing repo before installing into the generator. We need to ensure an end-to-end test of setting this up correctly before we can be sure that this configuration will work with a newly generated repo._** 

## Exit criteria 

This issue will be considered closed once a new repo created from the generator comes complete with these updates removing Synk and CodeQL, and installing SonarQube.

**Comment by blackfalcon** [comment]
This work is delayed until we have tokens for SonarCube from DevSecOps. This was reviewed in a team refinement meeting today and this body of work is ready to be addressed as soon as the SonarCube token prereq is addressed.

**Comment by blackfalcon** [comment]
SonarQube secrets have been added to all repos as a org secret. This work is ready to be addressed.

**Comment by megrenn** [comment]
https://dev.azure.com/itsals/E_Retain_Content/_workitems/edit/1374993
